.github/workflows/release.yml

```diff
@@ -36,6 +36,8 @@ jobs:
 
     - name: Checkout code
       uses: actions/checkout@v4
+      with:
+        persist-credentials: false
```

Comment:
Have you tried running the release action on your fork with these changes? I have a vague recollection of trying to add this `persist-credentials` line once myself

Reply:
Ah, forgot to write [wrkflw](https://github.com/bahdotsh/wrkflw) doesn't seem to work for me, so I didn't test release build. Perhaps `softprops/action-gh-release@v2` requires the credentials?
I'll try tomorrow.

Reply:
Appears to work fine, there is a draft release at https://github.com/mati865/wild/releases/tag/untagged-721530ea245c31cf02d2 and the job passed: https://github.com/mati865/wild/actions/runs/18687086619